zingo-testutils/src/interrupts.rs

```diff
@@ -0,0 +1,9 @@
+use zingolib::lightclient::LightClient;
+
```

Comment:
Can you throw a doc-comment with a doc-testable examples clause on this?

Why is select called with `biased`?

Reply:
doc tests included